functions/index.js

```diff
@@ -19,7 +19,9 @@ exports.createNewTask = functions.https.onRequest((request, response) => {
         var estimatedTime = request.body.data.EstimatedTime;
         var status = request.body.data.Status;
         var category = request.body.data.Category;
-        var taskIdNumber = getIdNumber();
+        var sprintId = request.body.data.SprintId;
+        var getSprintId = "S" + sprintId;
+        var taskIdNumber = getTaskId();
```

Comment:
Please check getTaskId() method is not available in this file. Please add it.

Reply:
i will add in this page